CPT331.Data.Parsers/NswXmlParser.cs

```diff
@@ -37,19 +37,19 @@ protected override void OnParse(string fileName, List<Crime> crimes)
 				string offenceName = xmlNode.ChildNodes[1].InnerText.Trim();
 				string suboffenceName = xmlNode.ChildNodes[2].InnerText.Trim();
 
-				Console.WriteLine($"{localGovernmentAreaName}: {offenceName} ({suboffenceName})");
+				//	Console.WriteLine($"{localGovernmentAreaName}: {offenceName} ({suboffenceName})");
 
 				Offence offence = null;
-				LocalGovernmentArea localGovernmentArea = localGovernmentAreas.Where(m => (m.Name == localGovernmentAreaName)).FirstOrDefault();
+				LocalGovernmentArea localGovernmentArea = localGovernmentAreas.Where(m => (m.Name.ToUpper() == localGovernmentAreaName.ToUpper())).FirstOrDefault();
```

Comment:
Would `m.Name.Equals(localGovernmentAreaName, StringComparison.InvariantCultureIgnoreCase)` be nicer?